src/utilities/BatchTable.js

```diff
@@ -0,0 +1,68 @@
+import { BatchTableHierarchyExtension } from './BatchTableHierarchyExtension.js';
+import { FeatureTable } from './FeatureTable.js';
+
+export class BatchTable extends FeatureTable {
+
+	constructor( buffer, batchSize, start, headerLength, binLength ) {
+
+		super( buffer, start, headerLength, binLength );
+		this.batchSize = batchSize;
+
+		const extensions = this.header.extensions;
+		if ( extensions ) {
+
+			this.extensions = {};
```

Comment:
We should always set `this.extensions = {}` even if there are no extensions. The class surface should be consistent, I think.

Reply:
You're right, I added it.